### ISSUE TITLE
TabPFNEmbedding Documentation and Usecase Help

### ISSUE THREAD

**Issue by TomatoTurtlez** [PriorLabs/tabpfn-extensions#92]
Hello,

We are using 'from tabpfn_extensions.embedding import TabPFNEmbedding to get embedding data using the following:

train_embeddings = embedding_extractor.get_embeddings(X_train, y_train, X_train, data_source='train')
test_embeddings = embedding_extractor.get_embeddings(X_train, y_train, X_test, data_source='test')

I have a question on what data_source is and how it's working in the background. I could not understand its purpose and function by reading the source code/documentation.

Also, if I have n_estimators = 8, is there a recommended approach for combining the 8 different embedding datasets into a single dataset for further analysis? In your examples, you use n_estimators = 1 to bypass this problem.

**Comment by eliott-kalfon** [comment]
Hi @TomatoTurtlez, we have just simplified our interface around embeddings, it was a bit confusing. Does this [documentation page](https://docs.priorlabs.ai/capabilities/embeddings) answer your question?

For your use case, I would recommend using the [cross-validated robust embeddings](https://docs.priorlabs.ai/capabilities/embeddings#1-out-of-fold-robust-embeddings-—-recommended).

Where they come from? Through cross-validation, we generate out-of-fold embeddings for the training data. This way, the embeddings for train and test live in the same space.

Using the vanilla embeddings, the training set embeddings would contain label information. Happy to help further if you have questions. cc @adrian-prior

**Comment by adrian-prior** [comment]
@TomatoTurtlez, I am closing the issue for now as it looks resolved to me. Feel free to reopen it, if still questions remain.
